Gems/Atom/Feature/Common/Assets/Materials/Types/MaterialInputs/ParallaxPropertyGroup.json

```diff
@@ -93,6 +93,19 @@
                 "name": "o_parallax_enablePixelDepthOffset"
             }
         },
+        {
+            "name": "shadowFactor",
+            "displayName": "Shadow Factor",
+            "description": "Can be used to reduce the impact of shelf-shadowing on surfaces with pixel depth offset (PDO), to reduce shadow acne or other artifacts.",
```

Comment:
"self-shadowing"

Reply:
I'd recommend describing how the range behaves (the significance of 0 and 1 here are somewhat nebulous, although it's clear from the documentation on `AdjustParallaxShadowCaster` itself).